### PR TITLE
Vip 626 add version flag

### PR DESCRIPTION
**Pull request by dwxander**
Adds version flag `-v`/`--version` to overwrite spec version upon output

### CODE REVIEW

File: src/vip_feed_format_converter/core.clj [hunk 1]
```diff
@@ -51,6 +52,7 @@
   (dissoc ctx :input))
 
 (defn set-handlers [ctx]
+  (println ctx)
```
Comment (danny-pease) [inline src/vip_feed_format_converter/core.clj:55]: opt - drop this line

**Review by danny-pease — APPROVED** [review]
this looks great, optional suggestion to remove the println

**Review by alice-dw — APPROVED** [review]
👍

**Review by danny-pease — APPROVED** [review]
🔢

**Pull request merged**